extension/source/Confirm/Confirm.tsx

```diff
@@ -76,13 +76,17 @@ const Confirm: FunctionComponent = () => {
           <div className="mt-4 flex justify-end text-body self-center gap-3">
             {current + 1} of {tx.actions?.length}
             <div
-              className="bg-grey-400 rounded-md p-1 hover:bg-grey-500 cursor-pointer"
+              className={[
+                'bg-grey-400 rounded-md p-1 hover:bg-grey-500 cursor-pointer',
```

Comment:
This is interesting. 
I have always used classNames in a single string but now I see the merit of breaking down each name as an array member to increase readability. It will also help when the classname is a computed value. 

WDYT about opening a new issue to change this everywhere in the codebase for consistency? I am really not comfortable with a single long string concatenated with itself :P 
It will a tedious thing though.